run/system-package/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "graphviz-web",
+  "version": "1.0.0",
+  "description": "Demonstrates a Cloud Run service which provides a CLI tool over HTTP.",
+  "main": "index.js",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "scripts": {
+    "start": "node index.js",
+    "test": "mocha test/app.test.js"
+  },
+  "author": "",
```

Comment:
I don't this this empty property is useful. Could be deleted.

Same with `license`.
We already have this property here:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/package.json#L5

Reply:
If this is standard, sure. Seems like these would be explicit in every package.json, along the same reasoning that every file has a license header. Removing for now.